### PR TITLE
Python additions

### PR DESCRIPTION
**Pull request by JamesPino**
1. AMLData class
Addition of python data class, enabling all input omics data to be handled in one initiation. The thought it to be able to build off of a single data class that can be reused throughout analysis and code. Examples on how to use in python & R can be found in the notebooks as well as the README.MD. 
``` python
from pybeataml.load_data import AMLData
data = AMLData()
# get trainable dataset in patient x feature format
source = ['proteomics', 'rnaseq'] # subset of data types
drug_name = 'Sorafenib' 
subset = data.get_trainable_data(source, drug_name)

# get specific data sets from class (simple convention for attributes)
mutations = data.wes
proteomics = data.proteomics
phospho = data.phospho
rna = data.rna

#subset data by datatype
# Long table format
rna_protein = data.subset_flat(['rnaseq', 'proteomics'])
# pivoted to be sample x label
rna_protein = data.subset(['rnaseq', 'proteomics'])
```

```R
library(reticulate)

reticulate::use_condaenv("beatAML_env_37", required = TRUE)
# assumes path 
reticulate::source_python("load_data.py")
# load data class
data <- AMLData()

# can access functions with $
mat <- data$all_data 

# same as python examples above, but use $
phospho <- data$phospho
rna <- data$rna

# function calls are the same as above as well
rna_protein <- data.subset(['rnaseq', 'proteomics'])
rna_protein <- data.subset_flat(['rnaseq', 'proteomics'])
```

2. Regression model construction
Created models folder, with various scripts for model creation. There is a main python script that runs through GBT, LASSO, ElasticNet, SVR models for all drugs and data combos. There is a Jupyter notebooks (described in README.MD) that explores and plots the output. I think these will go into a separate folder once the repos folder structure is further ironed out. 

3. Enrichment analysis
Basically simple examples of using MAGINEs enrichment analysis capabilities.  Right now the meta_gene_enrichment.ipynb uses the EnrichR module of MAGINE and does enrichment analysis and creates annotated set networks. Can still refactor this out to enable more streamlined analysis, but for now I think this gets across what it can do with our current gene set lists.

### CODE REVIEW

**Comment by JamesPino** [comment]
@sgosline might look like a large PR, but it is pretty self contained and shouldn't sprawl out to break anything. Before you merge if you want more specific examples of incorporation of the data class or anything else, just let me know and I can get right on it.
